src/erlfmt_scan.erl

```diff
@@ -147,6 +147,10 @@ read_rest(#state{scan = _Scan, inner = IO, loc = _Loc, buffer = Buffer}) ->
         {error, Reason} -> {error, {0, file, Reason}}
     end.
 
+read_rest(undefined, Data) ->
```

Comment:
Still struggling to figure out how read_rest receives an `undefined` atom in the first place.